securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -166,10 +166,14 @@ keycloak.init({ flow: 'hybrid' })
 The JavaScript adapter depends on Base64 (window.btoa and window.atob) and HTML5 History API.
 If you need to support browsers that do not have these available (for example, IE9) you need to add polyfillers.
 
-Example polyfill libraries:  
+Example polyfill libraries:
 
 * https://github.com/davidchambers/Base64.js
-* https://github.com/devote/HTML5-History-API        
+* https://github.com/devote/HTML5-History-API
+
+If available, the JavaScript adapter will use native Promise instances as return values from functions documented as returning promises.  To retain backwards compatibility, the `success()` and `error()` functions from previous versions of the adapter are retained.  These remain available whether or not native Promises are not provided by the browser.  Promise API polyfills are available for browsers without support:
```

Comment:
You can remove the second 'not' to make this sentence read better: "These remain available whether or not native Promises are not provided by the browser."